test/new-e2e/tests/otel/utils/pipelines_utils.go

```diff
@@ -783,6 +807,17 @@ func getContainerTags(t *testing.T, tp *trace.TracerPayload) (map[string]string,
 	return getTagMapFromSlice(t, splits), true
 }
 
+func getLogTags(t require.TestingT, log *aggregator.Log) map[string]string {
+	tags := getTagMapFromSlice(t, log.Tags)
+	attrs := make(map[string]interface{})
+	err := json.Unmarshal([]byte(log.Message), &attrs)
+	require.NoError(t, err)
+	for k, v := range attrs {
+		tags[k] = fmt.Sprint(v)
+	}
```

Comment:
Just to make sure: the func is called getLogTags but here is looping through the log message?

Reply:
Good point, just renamed to `getLogTagsAndAttrs`